README.md

```diff
@@ -21,38 +35,31 @@ To work on this project you should have:
 -   Docker (optional)
 Once you have the prerequisite software installed you can proceed to setup this application.
 
-### Node and Express setup
+## 2.1. Node and Express setup
 This application is designed to work as an API driven by Express. To setup your environment first you must install all required dependencies by running the following command from the root of your project directory:
 ```
 npm install
 ```
 Once all dependencies are installed you will need to setup some environment variables to interact with your database and application. 
 
-### Environment variables
+## 2.2. Environment variables
 You will need to set some local environment variables to run this application. This is true even if you're using Docker.
 ```
 touch .env
 echo 'NODE_ENV=local
 PORT=<your port>
-DATABASE_HOST=<your database host>
-DATABASE=<your database name>
-DATABASE_USER=<your database user>
-DATABASE_PASSWORD=<your database password>
 DATABASE_SCHEMA=<your database schema>
 JWT_KEY=<your secret JWT seed phrase or key>
 DATABASE_URL_DEV=<your connection string based on above variables>
 ' >> ./.env
 ```
 
-And example of the `DATABASE_URL_DEV` would be `DATABASE_URL_DEV=postgres://postgres:.@localhost:5432/postgres`
+And example of the `DATABASE_URL` would be `DATABASE_URL=postgres://user:pass@example.com:5432/dbname`

```

Comment:
DATABASE_URL_DEV in the prior .env section and DATABASE_URL here. Additionally, there are no variables associated with user, pass, or dbname in the prior section (as they were removed), are they meant to be defined for the first time here? This seems unclear if someone hasn't seen these variables declared previously.